front/src/components/controls/mmt-avatar/mmt-avatar.component.html

```diff
@@ -0,0 +1,8 @@
+<v-avatar :size="size">
+  <img v-if="src" :src="src" />
+  <v-icon v-else
+          :size="size"
+          dark>
+    mdi-account-circle
+  </v-icon>
+</v-avatar>
```

Comment:
do we add image for avatar without photo?

Reply:
I've add default-avatar.png